src/main/webapp/add-course.html

```diff
@@ -4,49 +4,88 @@
-                        <legend>How many units?</legend>      
-                        <input type="radio" name="num-units" value="1">1<br>      
-                        <input type="radio" name="num-units" value="2">2<br>      
-                        <input type="radio" name="num-units" value="3">3<br>      
-                    </fieldset> 
-                    <label for="term">Term:</label>
-                    <select id="term" name="term">
-                        <option value="Fall 2020">Fall 2020</option>
-                        <option value="Summer 2020">Summer 2020</option>
-                        <option value="Spring 2020">Spring 2020</option>
-                        <option value="Winter 2020">Winter 2020</option>
-                        <option value="Fall 2019">Fall 2019</option>
-                        <option value="Summer 2019">Summer 2019</option>
-                        <option value="Spring 2019">Spring 2019</option>
-                        <option value="Winter 2019">Winter 2019</option>
-                        <option value="Fall 2018">Fall 2018</option>
-                    </select>
-                </div>
-                <textarea name="school-name" id="school-name" placeholder="School Name" maxlength=1500></textarea>
-                <input class="course-button" id="add-button" type="submit" />
-            </form>
-            <button class="course-button" id="add-to-search" onclick="redirect('index.html')">Search for courses</button>
-            <button class="course-button" id="private-logout-button" onclick="signOutPrivate()">Sign out</button>
+        <div class="container-fluid bg-light p-5">
+            <div id="private-login-box">
+                <div class="g-signin2" id="private-login-button" data-onsuccess="signInPrivate"></div>
+                <div id="private-login-message"></div>
+            </div>
+            <div class="hidden container-xl my-4" id="private-class-info">
+                <form id="course-form" onsubmit="addCourse()">
+                    <h1 class="p-4 text-center">Enter course info:</h1>
+                    <div class="form-row" id="class-textarea">
+                        <div class="form-group col-md-6">
+                            <input type="text" class="form-control" name="course-name" id="course-name" placeholder="Class Name" maxlength=1500></textarea>
+                        </div>
+                        <div class="form-group col-md-6">
+                            <input type="text" class="form-control" name="prof-name" id="prof-name" placeholder="Professor Name" maxlength=1500></textarea>
+                        </div>
+                    </div>
+                    <div class="form-row">
+                        <div class="form-group col-md-4" id="class-selectboxes">
+                            <label for="unit-choice">How many units?</label>
+                            <select id="num-units" name="num-units" class="form-control">
```

Comment:
does this still not mean that units only go up to 3?

Reply:
Maybe just change it to a input field with type num and minimum of 1

Reply:
I'm going to do this in another PR, since this one is only styling.